python_ta/checkers/constant_test_checker.py

```diff
@@ -0,0 +1,55 @@
+"""
+checker for using constant value in a conditional statement
+"""
+import astroid
+
+from pylint.interfaces import IAstroidChecker
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import check_messages
+
+
+class UsingConstantTestChecker(BaseChecker):
+
+    __implements__ = IAstroidChecker
+
+    name = 'using-constants-test'
+
+    msgs = {'W7777': ('using a constant value in a conditional statement'
```

Comment:
Please use the numbering `E99XX` for your checkers (this helps me keep track of custom checkers, at least for now).